### DISCUSSION TITLE
무한 스크롤 훅의 위치에 대한 질문

### DISCUSSION THREAD

**Discussion by DongjaJ** [Anifriends/Anifriends-Frontend#107] [Q&A]
무한 스크롤은 범용적으로 사용할 것 같아요! 그래서 shared나 apps/shelter로 이동해도 괜찮지 않을까 싶은데 어떤가요

**Comment by sukvvon** [comment]
찬성입니다!! 🙋🏻‍♂️

**Reply by DongjaJ** [reply]
우선은 지금 작업하는 봉사자 리뷰 페이지 브랜치에서 apps/shelter/hooks로 이동시켜 놓겠습니다

**Reply by Eosdia** [reply]
좋아요~~
